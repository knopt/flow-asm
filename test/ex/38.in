1
49
8.792866267100244
0.7226383623947896 
9.43650950968631 
-3.074781496121104 
4.569720415919418 
-4.448645342956292 
7.2163149273165885 
-6.391849727259416 
-0.5789326016041123 
-8.424547037993687 
-2.018730899662426 
2.987664933212095 
-4.522012009112279 
-7.342596941389791 
-4.148162892584287 
5.702099448301475 
8.52258197469325 
-6.0361062072527165 
-8.137927597633855 
9.671078705237598 
-9.498846798710638 
1.7181627878549754 
-0.30636632906979777 
2.005474693837776 
-5.008593493540148 
-3.6354909260246053 
-9.472330490185982 
3.589813441193396 
-0.4192793314371883 
-6.801705430990839 
3.3461910500680165 
4.309485417777342 
-1.0233620989066061 
-1.3560375491601544 
-4.711918274795002 
9.189004311996335 
3.3310806206625934 
-1.3153416249559982 
7.9405137738729294 
6.06719833612453 
8.107249388364615 
-1.0455598771197376 
5.689776479868273 
5.771091536213081 
-6.082970522130388 
1.0098985756038115 
-9.415668869685794 
-5.348603685974005 
-0.6052586641559294 
9.140312520617972 

3.060999229809658 -7.781764870312529 2.015679091929279 -2.1969697922852305 0.49314482811627514 8.49911295890109 4.242193161909789 3.6613797672172765 3.0313464605117897 -0.09711942180681277 -3.0051604810361248 -3.6611830150401232 2.802388861324223 -5.585586894175261 8.484510309739616 3.010691897534601 -5.301082624119918 -5.205564609490523 7.828082120464334 9.290037473765715 3.897531737343714 4.78488271089029 3.891414960531648 3.264532616290838 6.120363904261147 -8.724322235688087 -2.438436393648531 -3.891743873512019 7.630710109401349 -4.765617004066504 1.3613437290611117 3.6477432796301184 -9.898200543464563 9.556213770188183 -4.505998436557245 -0.40022099223977214 -5.243449339364288 -8.323297605021054 -7.808777135509861 -0.8113554897903903 -2.1960099411765466 6.108536017322898 4.490607961460814 2.297550658183212 4.610931690019463 7.208985372881184 7.419621224571863 0.3826968787491296 -4.783012748493376 
