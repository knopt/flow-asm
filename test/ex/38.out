23.12535611398152 
-449.9008655032437 
185.24275491534829 
-259.7453764862469 
354.5161636854112 
-288.99441166770504 
477.2097881169953 
-9.255859366308494 
397.1084111131626 
38.30393772069306 
-245.34512873376016 
122.01765114687801 
182.56949870157962 
213.92924628004175 
-154.56075088148086 
-314.6398474213545 
196.80854874694796 
358.05017992081486 
-465.84372563332005 
693.0434901719764 
-2.006276816686448 
156.46427133249082 
-27.903486328230596 
317.59518701200903 
34.67595004627458 
362.2336072006626 
-373.58176367321073 
1.2077043665316995 
308.9394661935003 
-128.51812440019793 
-162.5890358552701 
26.9480189532889 
36.89876810717988 
228.69096158940843 
-366.0541683466973 
-163.1302685858183 
32.81271429851202 
-487.3285487967596 
-268.5486003409623 
-399.2728486424013 
193.5053625720656 
-129.01884016944825 
-138.00948032951783 
421.2057768881013 
-55.53407481014961 
535.5589504571759 
273.67888923020007 
85.892870701824 
-275.9816718458562 
